### PR TITLE
add sign in gutton and change css

### PR DESCRIPTION
**Pull request by Gugo-1991**

### CODE REVIEW

File: src/header/header/loginCreator.js [hunk 1]
```diff
@@ -13,14 +14,15 @@ function LoginCreator() {
...
       />
-
       <input
         className="password"
         onChange={(e) => setPassword(e.target.value)}
         type="password"
         placeholder="Enter your password"
       />
       <Button value={"Login"} login={login} password={password} />
+      <h6>or</h6>
```
Comment (agularyan) [inline src/header/header/loginCreator.js]: Please remove this

File: src/modal/modalContent/signIn.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <button
+        onClick={() => dispatch(closeModal())}
+        type="button"
+        className="btn btn-outline-danger">
+        Cancel
+      </button>
+    </>
+  );
+}
+
```
Comment (agularyan) [inline src/modal/modalContent/signIn.js:24]: Sign up button should be same size as `Login` and green.

File: src/modal/modalContent/signIn.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <>
+      <div className="signiindiv">
+        <input className="signin" placeholder="name..." />
+        <input className="signin" placeholder="email..." />
+        <input className="signin" placeholder="password..." />
+        <input className="signin" placeholder="confirm password..." />
+      </div>
+
+      <button className="btn btn-outline-dark"> Create account </button>
```
Comment (agularyan) [inline src/modal/modalContent/signIn.js]: On click should add user with ability to sign in.

File: src/modal/modalContent/signIn.js (hunk 2 above)
Comment (Gugo-1991) [reply src/modal/modalContent/signIn.js:24]: ok

File: src/modal/modalContent/signIn.js (hunk 3 above)
Comment (Gugo-1991) [reply src/modal/modalContent/signIn.js]: ok

**Pull request merged**